src/components/alerts/LoginSuccess.tsx

```diff
@@ -15,80 +15,81 @@ const LoginSuccess = () => {
   const [progress, setProgress] = useState(0);
 
   useEffect(() => {
-    const interval = 10; // update every 50ms
-    const totalDuration = 3000; // 5 seconds in total
+    const totalDuration = 3000; // 3 seconds in total
+    const startTime = Date.now();
 
-    // Initialize the progress state to totalDuration
-    setProgress(totalDuration);
+    function updateProgress() {
+      const elapsed = Date.now() - startTime;
 
-    const timer = setInterval(() => {
-      setProgress((prev) => {
-        const nextValue = prev - interval;
-        if (nextValue <= 0) {
-          clearInterval(timer);
-          setShowAlert(false);
-          return 0;
-        }
-        return nextValue;
-      });
-    }, interval);
+      if (elapsed >= totalDuration) {
+        setProgress(0);
+        setShowAlert(false);
+      } else {
+        setProgress(totalDuration - elapsed);
+        requestAnimationFrame(updateProgress);
+      }
+    }
 
-    return () => clearInterval(timer);
+    requestAnimationFrame(updateProgress);
+
+    return () => {};
   }, []);
 
   return (
     <>
-      {showAlert && currentUser.userType === "student" && (
-        <Box
-          width={"50%"}
-          position="relative"
-          margin="auto"
-          display="block"
-          marginBottom={"10px"}
-        >
-          <Alert display={"flex"} width={"100%"} status="success">
-            <AlertIcon />
-            <AlertTitle mr={2}>Login Successful</AlertTitle>
-            <AlertDescription flex="1">
-              Welcome {currentUser.name}! Good Luck On Your Exam!
-            </AlertDescription>
-          </Alert>
-          <Progress
-            size="xs"
-            value={(progress / 3000) * 100} // Adjust the denominator to your total duration
-            position="absolute"
-            width="100%"
-            bottom="0"
-            colorScheme="green"
-          />
-        </Box>
-      )}
+      {showAlert &&
+        currentUser.userType === "student" && ( //student login success alert
```

Comment:
Btw usually its best practice to standardise the formatting tools used in a team to avoid big diffs that actually dont change anything i.e this file

Reply:
you'll have to explain this on monday